engine/flink/generic/src/test/scala/pl/touk/nussknacker/genericmodel/GenericItSpec.scala

```diff
@@ -233,23 +250,18 @@ class GenericItSpec extends FunSuite with BeforeAndAfterAll with Matchers with K
   }
 
   private lazy val creator = new GenericConfigCreator {
```

Comment:
These tests fail with serialization errors because `GenericConfigCreator` implements `Serializable`, and you define `creator` here as an inner class of class that's not serializable, and because of the very nature of inner classes it won't work (see: https://stackoverflow.com/questions/17804704/notserializableexception-on-anonymous-class). Just make sure you define the class somewhere outside, and it will work like a charm!

Reply:
I've allredy figured it out ;) But thanks for tip!